pyproject.toml

```diff
@@ -1,70 +1,60 @@
-[build-system]
-requires = ["setuptools >= 65", "setuptools-scm >= 7"]
-build-backend = "setuptools.build_meta"
-
-[project]
-name = "asyncio_mqtt"
-description = "Idiomatic asyncio wrapper around paho-mqtt"
+[tool.poetry]
+name = "aiomqtt-test"
+version = "0.0.0"  # Placeholder: Managed by poetry-dynamic-versioning
+description = "The idiomatic asyncio MQTT client, wrapped around paho-mqtt"
+license = "BSD-3-Clause"
+authors = ["Frederik Aalund <fpa@sbtinstruments.com>"]
```

Comment:
Add yourself here as well! :)

Let's just make `authors` the same as `maintainers`. That's how it is in practice.